test/k8s_service_service_account_template_test.go

```diff
@@ -22,30 +21,21 @@ func TestK8SServiceAccountCreateTrueCreatesServiceAccount(t *testing.T) {
 	t.Parallel()
 	randomSAName := strings.ToLower(random.UniqueId())
 
-	helmChartPath, err := filepath.Abs(filepath.Join("..", "charts", "k8s-service"))
-	require.NoError(t, err)
+	serviceaccount := renderK8SServiceAccountWithSetValues(
```

Comment:
I've created the new helper function similar to the others and refactored the tests here.